### PR TITLE
Added eclipse code format files

### PR DESCRIPTION
**Pull request by JPinzon01**
- Formatter: Manages code indentation, spaces, line breaking and line
  wrapping.
- Clean Up: Manages changes or corrections to code, such as eliminating
  unnecessary code, add missing annotations, remove trailing space,
  etc.
- Import organization: Manages the order of the packages in java files.
- Modifications in README.md to reflect the changes and minor corrections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,11 +28,17 @@ for Java, with the following changes (mandatory):
 
 * IntelliJ
   - Go to "Preferences" and them to "Coding Style" in IntelliJ.
```
Comment (magicDGS) [inline README.md]: Could you rephrase this as in the Eclipse setting: `Go to "Preferences" -> "Coding Style"` and remove IntelliJ, to have consistency.

File: README.md [hunk 2]
```diff
@@ -27,11 +28,17 @@ for Java, with the following changes (mandatory):
...
-  - Inport [intellij-java-magicdgs-style.xml](https://github.com/magicDGS/styleguide/blob/master/intellij-java-magicdgs-style.xml)
+  - Import [intellij-java-magicdgs-style.xml](https://github.com/magicDGS/styleguide/blob/master/intellij-java-magicdgs-style.xml)
+
+* Eclipse
```
Comment (magicDGS) [inline README.md:33]: IntelliJ have an option to import Eclipse styles. Did you check if this ones are working as expected if you do that?

**Comment by magicDGS** [comment]
One change in the README and a question to be answered. Squash commits and I will merge when addressed. Back to you, @JPinzon01.

File: README.md (hunk 2 above)
Comment (JPinzon01) [inline README.md:33]: It works, but many options are not set. Keep in mind that we need three files for eclipse, while only one of them can be imported as long as I know. The options in the other files have to be set manually.

Comment (magicDGS) [inline README.md:33]: Ok, it's fine like that.
